### ISSUE TITLE
Following PSR standards?

### ISSUE THREAD

**Issue by mnapoli** [rossriley/php-scalar-objects#9]
This is a suggestion: how about following PSR-1 and PSR-2 for code formatting? I see that you are already following PSR-4, that would help to contribute people taking this project seriously I believe.

**Comment by rossriley** [comment]
Yes. Good suggestion. I'll get started on fixing up the problems.

**Comment by davidstanley01** [comment]
I can handle it if you want.

**Comment by rossriley** [comment]
That would be gret.

**Comment by davidstanley01** [comment]
I'll start on it tomorrow
